mysql-test/suite/rocksdb/t/rocksdb_datadir.test

```diff
@@ -1,8 +1,13 @@
 --source include/have_rocksdb.inc
 
+if (!$ROCKSDB_LOAD_PATH) {
```

Comment:
Is it possible have include/have_rocksdb.inc passing and $ROCKSDB_LOAD_PATH not set?